plugin/ochttp/propagation/tracecontext/propagation.go

```diff
@@ -24,29 +24,23 @@ import (
 
 	"go.opencensus.io/trace"
 	"go.opencensus.io/trace/propagation"
-	"go.opencensus.io/trace/tracestate"
-	"regexp"
+	httpTs "go.opencensus.io/trace/propagation/http/tracestate"
```

Comment:
is this a typo?

Reply:
earlier I had trace/tracestate and trace/propagation/http/tracestate. So I had httpTs as differentiator. 
Fixed it.